blockchain/src/history/light_history_store.rs

```diff
@@ -294,23 +294,26 @@ impl HistoryInterface for LightHistoryStore {
         _tx_hash: &Blake2bHash,
         _txn_option: Option<&TransactionProxy>,
     ) -> Vec<HistoricTransaction> {
-        todo!()
+        log::warn!("Get history tx by hash is unsupported in the light history store");
+        vec![]
```

Comment:
Hmm. Not sure just retuning an empty list is the way to go.
If I make this request to a remote full node via RPC, I don't see this log message.

How much work is it to change to a `Result` return type?